src/deserialization.hpp

```diff
@@ -404,4 +446,55 @@ Status deserializePredictRequest(
     }
     return status;
 }
+template <class TensorProtoDeserializator, class Sink>
+Status deserializePredictRequest(
+    const InferenceRequest& request,
+    const tensor_map_t& inputMap,
+    Sink& inputSink, bool isPipeline) {
+    OVMS_PROFILE_FUNCTION();
+    Status status;
+    for (const auto& pair : inputMap) {
+        try {
+            const auto& name = pair.first;
```

Comment:
You can use cpp17 feature `for (const auto& [name, tensorInfo] : inputMap) {` to save 2 lines of code